wgs/community-membership.md

```diff
@@ -8,10 +8,12 @@ Responsibilities for most roles are scoped to these subprojects.
 
 | Role | Responsibilities | Requirements | Defined by |
 | -----| ---------------- | ------------ | -------|
-| member | active contributor in the community | sponsored by 2 reviewers.  multiple contributions to the project. | Kubernetes GitHub org member. |
-| reviewer | review contributions from other members | history of review and authorship in a subproject | [OWNERS] file reviewer entry. |
-| approver | approve accepting contributions | highly experienced and active reviewer + contributor to a subproject | [OWNERS] file approver entry|
-| subproject owner | set direction and priorities for a subproject | demonstrated responsibility and excellent technical judgement for the subproject | [wgs.yaml] subproject [OWNERS] file *owners* entry |
+| Member | Active contributor in the community | Sponsored by 2 reviewers.  Multiple contributions to the project. | GitHub org member. |
+| Scribe | Ensure important information is represented in working group notes | Membership | Write access to WG documentation | 
```

Comment:
What's Scribe?

Reply:
This is defined in the section below, perhaps I should link each title to the definition of the role below?